### DISCUSSION TITLE
Regular Expression \p{N} and \p{L}

### DISCUSSION THREAD

**Discussion by janic0** [vercel/next.js#32752] [Help]
### What version of Next.js are you using?

 v10.2.3

### What version of Node.js are you using?

v14.17.0 (ts-node on v10.0.0)

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

I have two regular expressions that use unicode character classes. `\p{N}` matches every number and `\p{L}` matches every character in the `character`class. 

Example
`/^[\p{L}\p{N}\.]{5,}$/u`

[module with the regular expressions](
https://gist.github.com/janic0/a8ec70c53a8b7cbede982b8f3376a4d5)

when opening the page, it fails to compile throwing this error:

`Error: pages/test.js: Unknown property: L`
`    at Array.map (<anonymous>)`
`    at Array.map (<anonymous>)`

### Expected Behavior

I have tried to use this regular expression in `vanilla js` as well as `React` and it **worked** on both of them. 

### To Reproduce

Just use the following regular expression somewhere in a `React component`. 

`/^[\p{L}\p{N}]+$/u`

**Comment by andrewsmith** [comment]
There's an open bug for this: https://github.com/vercel/next.js/issues/19303
